components/service/experiments/KintoSchema.md

```diff
@@ -0,0 +1,375 @@
+# Kinto Schema
+
+This document contains the information about the Kinto schema and UI schema needed to run dev experiments
+on the "Dev" Kinto instance located at https://kinto.dev.mozaws.net.
+
+## JSON Schema
+
+  ```JSON
+  {
```

Comment:
It's a small thing, but could you dedent the code blocks so that there isn't any space her and on the fence lines?

Reply:
Certainly! I didn't see this before but it must have been an artifact of the copy/paste from the gist. Thanks for pointing it out.